frontend/apps/picasso/pages/crowdloan-rewards/claim.tsx

```diff
@@ -94,40 +102,66 @@ export const ClaimLoanPage = () => {
       accounts
     );
 
-  const nextStep = useCrowdloanRewardsStepGivenConnectedAccounts(
-    selectedAccount?.address,
-    account?.toLowerCase()
-  );
+  useEffect(() => {
+    if (parachainApi && account) {
+      parachainApi.query.crowdloanRewards
+        .rewards<Option<PalletCrowdloanRewardsModelsReward>>({
+          ethereum: account.toLowerCase(),
+        })
+        .then((result) => {
+          if (result.isSome) {
+            const total = fromChainIdUnit(result.unwrap().total.toString());
+            const claimed = fromChainIdUnit(result.unwrap().claimed.toString());
+
+            setEthClaimable(total.minus(claimed));
+            setEthClaimed(claimed);
+            setEthTotal(total);
```

Comment:
I'm a bit confused by the naming here. We're not claiming eth, we're claiming PICA.